crates/rooch-framework/sources/coin.move

```diff
@@ -239,6 +239,17 @@ module rooch_framework::coin {
         }
     }
 
+    /// Return coin info handle
+    public fun coin_info_handle(ctx: &StorageContext): Option<ObjectID> {
+        if (account_storage::global_exists<CoinInfos>(ctx, @rooch_framework))
```

Comment:
I think this should be ensured via the Genesis transaction, so it should always exist.

Reply:
done